pygotham/frontend/templates/talks/proposal.html

```diff
@@ -19,6 +19,10 @@
         {{ wtf.horizontal_field(form.duration, placeholder='Duration') }}
         {{ wtf.horizontal_field(form.abstract, placeholder='Abstract') }}
         {{ wtf.horizontal_field(form.outline, placeholder='Outline') }}
+        {% for invite_form in form.speaker_invites %}
+          {{ invite_form.hidden_tag() }}
+          {{ wtf.horizontal_field(invite_form.invited_email, placeholder='your.partner@in.crime') }}
```

Comment:
The placeholder should be consistent with the others, something like `Email` or `Co-presenter email`.